features/preferences/impl/src/main/res/values/localazy.xml

```diff
@@ -6,6 +6,8 @@
   <string name="screen_advanced_settings_developer_mode">"Developer mode"</string>
   <string name="screen_advanced_settings_developer_mode_description">"Enable to have access to features and functionality for developers."</string>
   <string name="screen_advanced_settings_rich_text_editor_description">"Disable the rich text editor to type Markdown manually."</string>
+  <string name="screen_advanced_settings_send_read_receipts">"Read receipts"</string>
+  <string name="screen_advanced_settings_send_read_receipts_description">"If turned off, your read receipts won\'t be sent to anyone. You will still receive read receipts from other users."</string>
```

Comment:
> You will still receive read receipts from other users.

Except if they also disabled the public RR :)

Reply:
I think that's kind of implicit, but please let's not bring back the discussion on the text used here, I already had enough of it 😅 .

Reply:
Yes, sure, I was sort of kidding :).

It reminds me the old days where mobile phone users could pay an option to get incoming call number displayed, and others could pay an option to hide their phone number when they're doing call :)